research/format.markdown

```diff
@@ -0,0 +1,138 @@
+## Draft 2 to Draft 3:
+
+> Behaviour: Not required to validate → MAY validate(but aren’t required to)
+
+**\-**  **street-address** : This should be a street address.
+
+**\-**  **locality** : This should be a city or town.
```

Comment:
Why do you have backslashes before the hyphens? Also, why are you bolding the hyphens?

Reply:
In any case, these lists feel like tables with two columns. Let's convert them into tables? Also, by reading this, its still unclear to me whether these are formats from Draft 2, from Draft 3, or the ones from Draft 2 that need to be updated, or the ones that were deleted, etc. Can you clarify?

Reply:
Ah, I just realized you have minus and plus symbols. I think it might be too subtle for a casual reader. If we turn these into tables, then on one column you can very literally say "Added", "Removed", or "Changed" (which not sure how you are representing this latter one right now)?

Reply:
I felt the table growing wide wouldn't be that good. So, I tried to have + and -. Is there a way I can haved green and red coloured + and -. and with bigger size. If this is possible I guess we can go with signs. Else table will be a better choice

Reply:
> Is there a way I can haved green and red coloured + and -.

I don't think so. Let's try the table. I think we can still make them reasonably wide that exploring it will not be a problem, but we'll see